mainline-alpine/entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+# Configure docroot.
+if [ -n "$NGINX_DOCROOT" ]; then
```

Comment:
As I have mentioned before, this variable should be nginx independent. Maybe just `DOCROOT`?